en/CommandLine.md

```diff
@@ -3,13 +3,12 @@ title: Command line use and options
 helpCategories: ["General"]
 ---
 
-# Command line use and options
+## Command line use and options
```

Comment:
First level heading is OK here. --> If the linter reports an error here, the linter is wrong.